language_service/src/state/tests.rs

```diff
@@ -1239,56 +740,218 @@ async fn update_doc_updates_project() {
             }
         "#]],
         &expect![[r#"
-            project/qsharp.json: SourceMap {
-                sources: [
-                    Source {
-                        name: "project/src/other_file.qs",
-                        contents: "namespace Foo { @EntryPoint() operation Main() : Unit {} }",
-                        offset: 0,
-                    },
-                    Source {
-                        name: "project/src/this_file.qs",
-                        contents: "namespace Foo { we should see this in the source }",
-                        offset: 59,
-                    },
-                ],
-                common_prefix: Some(
-                    "project/src/",
-                ),
-                entry: None,
-            }
+            project/qsharp.json: [
+              "project/src/other_file.qs": "namespace Foo { @EntryPoint() operation Main() : Unit {} }",
+              "project/src/this_file.qs": "namespace Foo { we should see this in the source }",
+            ],
         "#]],
         &expect![[r#"
             [
-                (
-                    "project/src/this_file.qs",
-                    Some(
-                        1,
-                    ),
+              uri: "project/src/this_file.qs" version: Some(1) errors: [
+                syntax error
+                  [project/src/this_file.qs] [we]
+              ],
+            ]"#]],
+    );
+}
+
+#[tokio::test]
```

Comment:
Here are the 2 new test cases, in case you have trouble finding it in the noise.